tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -0,0 +1,434 @@
+			'en_NZ'		=> __( 'New Zealand', 'tribe-extension' ),
+			'pt_PT'		=> __( 'Portugal', 'tribe-extension' ),
+			'en_SG'		=> __( 'Singapore', 'tribe-extension' ),
+			'en_GB'		=> __( 'United Kingdom', 'tribe-extension' ),
+			'en_US'		=> __( 'United States', 'tribe-extension' ),
+		);
+
+		$array = $this->append_array_keys_to_values( $array );
+
+		return $array;
+	}
+
+	private function append_array_keys_to_values( $array = '' ) {
+		if ( ! is_array( $array ) || empty( $array ) ) {
+			return false;
+		}
+
+		$new_array = array();
+
+		foreach ( $array as $key => $value ) {
+			$new_array[$key] = sprintf( '%s (%s)', $value, $key );
+		}
+
+		return $new_array;
+	}
+
+	/**
+	 * Get user-selected Eventbrite locale
+	 *
+	 * @access private
+	 * @return string
+	 */
+	private function get_chosen_eb_locale() {
+		$locale = tribe_get_option( $this->opts_prefix . 'locale' );
+
+		if ( empty( $locale )
+			|| ! is_string( $locale )
+			|| ! array_key_exists( $locale, $this->eb_locale_country_name_array() )
+		) {
+			$locale = 'en_US';
+		}
+
+		return $locale;
+	}
+
+	/**
+	 * Get Eventbrite currency from specified locale
+	 * This list is hand-crafted from noting the currency specified in the calculator at https://www.eventbrite.com/fees/ (per TLD) after selecting each region from Eventbrite's Region Selector. Could just play with PHP's https://secure.php.net/localeconv but decided against it to avoid changing system settings.
+	 * @access private
+	 * @return string
+	 */
+	private function get_currency_from_eb_locale() {
+		$locale = $this->get_chosen_eb_locale();
+
+		$locale_currencies = array(
+			'es_AR'		=> 'ARS',
+			'en_AU'		=> 'AUD',
+			'pt_BR'		=> 'BRL',
+			'en_CA'		=> 'CAD',
+			'fr_CA'		=> 'CAD',
+			'de_DE'		=> 'EUR',
+			'es_ES'		=> 'EUR',
+			'fr_FR'		=> 'EUR',
+			'en_HK'		=> 'HKD', // EB's fees calculator does not display
+			'en_IE'		=> 'EUR',
+			'it_IT'		=> 'EUR',
+			'nl_NL'		=> 'EUR',
+			'en_NZ'		=> 'NZD',
+			'pt_PT'		=> 'EUR',
+			'en_SG'		=> 'SGD', // EB's fees calculator does not display
+			'en_GB'		=> 'GBP',
+			'en_US'		=> 'USD',
+		);
+
+		$currency = $locale_currencies[$locale];
+
+		return $currency;
+	}
+
+	/**
+	 * Get Eventbrite Top-Level Domain (TLD) from specified locale
+	 * This list is hand-crafted from noting the domain change after selecting each region from Eventbrite's Region Selector.
+	 * @access private
+	 * @return string
+	 */
+	private function eb_locale_to_tld() {
+		$locale = $this->get_chosen_eb_locale();
+
+		$locale_tlds = array(
+			'es_AR'		=> '.com.ar',
+			'en_AU'		=> '.com.au',
+			'pt_BR'		=> '.br',
+			'en_CA'		=> '.ca',
+			'fr_CA'		=> '.ca', // it is the same!
+			'de_DE'		=> '.de',
+			'es_ES'		=> '.es',
+			'fr_FR'		=> '.fr',
+			'en_HK'		=> '.hk',
+			'en_IE'		=> '.ie',
+			'it_IT'		=> '.it',
+			'nl_NL'		=> '.nl',
+			'en_NZ'		=> '.co.nz',
+			'pt_PT'		=> '.pt',
+			'en_SG'		=> '.sg',
+			'en_GB'		=> '.co.uk',
+			'en_US'		=> '.com',
+		);
+
+		$tld = $locale_tlds[$locale];
```

Comment:
Not per https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage

And I didn't see "you should add spacing in this context" mentioned here: https://moderntribe.github.io/products-engineering/php/

Is there somewhere else to look? Is there something I can import to PhpStorm to reformat code differently than the WordPress defaults it already comes with?